package.json

```diff
@@ -20,7 +20,7 @@
     "@ethersproject/abstract-signer": "^5.1.0",
     "@ethersproject/providers": "^5.1.0",
     "@pooltogether/v4-autotask-lib": "^0.1.0-beta.6",
-    "@pooltogether/v4-pool-data": "1.5.0-beta.2",
+    "@pooltogether/v4-pool-data": "1.8.0-beta.3",
```

Comment:
Bump to [1.8.0-beta.4](https://www.npmjs.com/package/@pooltogether/v4-pool-data/v/1.8.0-beta.4)